task-1/src/constants/colors.js

```diff
@@ -0,0 +1,6 @@
+export const MAIN_COLOR = "#7C005A";
+export const TITLES_BACKGROUND_COLOR = "#F39DDC";
+export const ADD_INFO_BACKGROUND_COLOR = "#FDCCEF";
+export const MAIN_POINTER_COLOR = "#37A110";
+export const TITLES_POINTER_BACKGROUND_COLOR = "#A0F08E";
+export const ADD_INFO_POINTER_BACKGROUND_COLOR = "#C3F0B4";
```

Comment:
newline

Reply:
It's a good practice to have color variables named  like PRIMARY_COLOR and SECONDARY_COLOR, so that you color pallete is clear, the rest is fine